package.json

```diff
@@ -2,12 +2,10 @@
   "name": "surrealdb.js",
   "version": "0.3.1",
   "description": "Javascript driver for SurrealDB",
```

Comment:
Add fields
```
  "unpkg": "dist/browser/index.js",
  "jsdelivr": "dist/browser/index.js",
```
so cdn willchoose the correct file to serve when file is not specified.